db/src/main/java/net/bible/android/database/bookmarks/BookmarksDao.kt

```diff
@@ -35,6 +35,7 @@ import java.util.*
 const val orderBy = """
 CASE WHEN :orderBy = 'BIBLE_ORDER' THEN Bookmark.kjvOrdinalStart END,
 CASE WHEN :orderBy = 'BIBLE_ORDER' THEN Bookmark.startOffset END,
+CASE WHEN :orderBy = 'CREATED_AT_DESC' THEN Bookmark.createdAt END DESC,
```

Comment:
I believe this should be

```suggestion
CASE WHEN :orderBy = 'CREATED_AT_DESC' THEN Bookmark.createdAt DESC END,
```